README.md

```diff
@@ -137,7 +138,11 @@ withFailure
 withFlowDetermination
 ```
 
-And a function `composeInjections` to help crafting your own async action definitions.  
+There's currently documentation for the following:
+
+- [withSuccess](./src/injections/withSuccess/docs.md)
+
+And a function `composeInjections` to help crafting your own async action definitions. 
```

Comment:
I'd remove this line, since `composeInjections` now is being used by the middleware 😇